app-backend/db/src/main/scala/ObjectPermissions.scala

```diff
@@ -192,18 +193,23 @@ trait ObjectPermissions[Model] {
                     actionType: ActionType,
                     ownershipTypeO: Option[String] = None,
                     groupTypeO: Option[GroupType] = None,
-                    groupIdO: Option[UUID] = None): Option[Fragment] = {
+                    groupIdO: Option[UUID] = None,
+                    tableNameO: Option[String] = None): Option[Fragment] = {
+    val tableName: String = tableNameO match {
+      case Some(tableName) => s"${tableName}."
+      case _ => ""
+    }
     val ownedF: Fragment =
-      Fragment.const(s"owner = '${user.id}'")
+      Fragment.const(s"${tableName}owner = '${user.id}'")
     val visibilityF: Fragment =
-      createVisibilityF(objectType, actionType)
+      createVisibilityF(objectType, actionType, tableName)
     val sharedF: Fragment =
       Fragment.const(
         s"""ARRAY['ALL;;${actionType.toString}', 'USER;${user.id};${actionType.toString}']""")
     val inheritedF: Fragment =
       createInheritedF(user, actionType, groupTypeO, groupIdO)
     val acrFilterF
-      : Fragment = fr"array_cat(" ++ sharedF ++ fr"," ++ inheritedF ++ fr") && acrs"
+      : Fragment = Fragment.const("array_cat(") ++ sharedF ++ Fragment.const(",") ++ inheritedF ++ Fragment.const(s") && ${tableName}acrs")
```

Comment:
I'm not sure that using Fragement.const really adds anything here for the increased verbosity, since these are all uninterpolated strings.
My understanding is that we should only be using Fragment.const where we absolutely need to avoid escaping interpolated values because it's a possible sql injection site.

Reply:
We only need to `Fragment.const` when interpolating things that aren't SQL parameters: https://tpolecat.github.io/doobie-cats-0.4.2/08-Fragments.html.

So, I agree with this just using an `fr` here unless we're missing something.